From 94ae72948f1ed1a8bad1c13b4edc4b9189207e35 Mon Sep 17 00:00:00 2001
From: Tom Hochstein <tom.hochstein@nxp.com>
Date: Mon, 26 Jul 2021 09:06:26 -0500
Subject: [PATCH] Makefile: Honor LDFLAGS from toolchain

The LDFLAGS variable is standard from the toolchain and should be
honored. Yocto detects the problem with a QA check:

ERROR: imx-vpu-hantro-daemon-1.1.0-r0 do_package_qa: QA Issue: File /usr/bin/vsidaemon in package imx-vpu-hantro-daemon doesn't have GNU_HASH (didn't pass LDFLAGS?) [ldflags]

Upstream-Status: Pending

Signed-off-by: Tom Hochstein <tom.hochstein@nxp.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 9a775b0..e73ff47 100755
--- a/Makefile
+++ b/Makefile
@@ -22,7 +22,7 @@ OUT_FILE = vsidaemon
 DEC = nxp_enc_hw
 ENC = nxp_dec_hw
 LIBFILES = 
-LD_FLAGS = -lpthread -lm
+LD_FLAGS = $(LDFLAGS) -lpthread -lm
 
 PLATFORM ?= IMX8MP
 
-- 
2.17.1

